README.md

```diff
@@ -71,9 +79,9 @@ The following directories are automatically created within this container:
 - `archive` - successfully processed files
 - `quarantine` - unsuccessfully processed files
 
-## Payment batch file specification
-### SFI Pilot
-#### Example file
+### Payment batch file specification
+#### SFI Pilot
+##### Example file
 Filename: `SITIELM0001_AP_20210812105404541.dat`
 ```
 B^2021-08-12^2^200^0001^SFIP^AP
```

Comment:
Further down here we have a table of scheme name and codes, here's a full list courtesy of John which you can add:

https://github.com/DEFRA/ffc-pay-statement-constructor/pull/18/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R69

Will need to differentiate between SFI Pilot, SFI, Lump Sums, etc